tests/pdk/finfet_pdk/test_circuits_partial_routing.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+import json
+import shutil
+import textwrap
+from .utils import get_test_id, build_example, run_example
+from . import circuits
+
+from align.pdk.finfet import transistor_array
+
+cleanup = False
+
+@pytest.fixture
```

Comment:
@soneryaldiz Here is the monkeypatching I did and the tests I choose to keep. Let me know if we should have more or less.